Programmers/jiwoo/26_week/수식 최대화.py

```diff
@@ -0,0 +1,41 @@
+from itertools import permutations
+
+def operation(num1, num2, operator):
+    if operator == '+':
+        return num1 + num2
+    if operator == '-':
+        return num1 - num2
+    if operator == '*':
+        return num1 * num2
+    
+def calculate(exp, operator):
+    array = []
+    temp = ""
+    for i in exp:
+        if i.isdigit() == True:
```

Comment:
True 쓸 필요 없이 그냥 아래처럼 해도 동일합니다!
```suggestion
        if i.isdigit():
```